HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py

```diff
@@ -265,7 +265,7 @@
             "HLT_L1_TripleJet_VBF_v"
             ),
         recJetLabel  = cms.string("ak4PFJetsCHS"),
-        jetTagLabel  = cms.string("pfCombinedSecondaryVertexBJetTags"),
```

Comment:
The pfCombinedSecondaryVertexBJetTags had been recommended to us a year ago (#5684) and then were never checked again.
Thanks for updating it.